hugolib/config.go

```diff
@@ -132,4 +133,5 @@ func loadDefaultSettingsFor(v *viper.Viper) {
 	v.SetDefault("enableGitInfo", false)
 	v.SetDefault("ignoreFiles", make([]string, 0))
 	v.SetDefault("disableAliases", false)
+	v.SetDefault("SummaryLength", 200)
```

Comment:
As noted above: The default above should remain at 70 (changing that will break many many sites).